account_reconcile_filter/README.rst

```diff
@@ -0,0 +1,26 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Account Reconcile Filter
+========================
+This module was written to extend the functionality of the client filter.
```

Comment:
Please extend a the module description. Add informations about the functionality and what changes the module does.